### DISCUSSION TITLE
npm audit sometimes times out ??

### DISCUSSION THREAD

**Discussion by Abloume** [verdaccio/verdaccio#3286] [Q&A] [answered]
I switched registry still not effective, npm audit not always return data...

warn --- could not fetch audit: FetchError: request to https://registry.yarnpkg.com/-/npm/v1/security/audits failed, reason: socket hang up
0|verdaccio  |  http --- 500, user: null(x.x.x.x), req: 'POST /-/npm/v1/security/audits', bytes: 35494/0

warn --- could not fetch audit: FetchError: request to https://registry.yarnpkg.com/-/npm/v1/security/audits failed, reason: connect ETIMEDOUT 104.16.18.35:443

**Comment by juanpicado** [comment, marked as answer]
hi @Abloume , 🤔 this might depends of `npmjs` rather Verdaccio unless you find any hint we can improve, I can describe what Verdaccio does so far, just re-route the request to npmjs services, that's all, if they fail, Verdaccio fails, we don't do much aside of that. Maybe is a transitory issue you could check in your side more in deep.

**Reply by Abloume** [reply]
Thank you very much. I'll try something else

**Reply by Yukiniro** [reply]
Is the problem solved?
